numpy/lib/tests/test_histograms.py

```diff
@@ -298,6 +298,12 @@ def test_datetime(self):
         assert_equal(d_edge.dtype, dates.dtype)
         assert_equal(t_edge.dtype, td)
 
+    def test_signed_overflow_bounds(self):
+        arr = np.array([-2**15 + 4, 2**15 - 4], dtype=np.int16)
```

Comment:
Should really do this for all the affected signed types.

Reply:
Done